koku/masu/database/report_db_accessor_base.py

```diff
@@ -112,9 +113,32 @@ def _get_db_obj_query(self, table, columns=None):
                 query = table.objects.all()
             return query
 
+    def extract_context_from_sql_params(self, sql_params: dict):
+        return {
+            "schema": sql_params.get("schema"),
+            "start_date": sql_params.get("start") or sql_params.get("start_date"),
+            "end_date": sql_params.get("end") or sql_params.get("end_date"),
+            "provider_uuid": sql_params.get("source_uuid"),
+            "invoice_month": sql_params.get("invoice_month"),
+        }
+
+    def _prepare_and_execute_raw_sql_query(self, table, tmp_sql, tmp_sql_params=None, operation="UPDATE"):
+        """Prepare the sql params and run via a cursor."""
+        if tmp_sql_params is None:
+            tmp_sql_params = {}
+        LOG.info(
+            log_json(
+                msg=f"triggering {operation}",
+                table=table,
+                context=self.extract_context_from_sql_params(tmp_sql_params),
```

Comment:
You can also just splat in the kwargs.
```suggestion
                **self.extract_context_from_sql_params(tmp_sql_params),
```